base/src/main/java/io/spine/validate/Validate.java

```diff
@@ -43,11 +47,11 @@
 import static com.google.protobuf.TextFormat.shortDebugString;
 import static io.spine.util.Exceptions.newIllegalArgumentException;
 import static io.spine.util.Exceptions.newIllegalStateException;
-import static io.spine.validate.MessageValidator.validate;
 
 /**
  * This class provides general validation routines.
  */
+@SuppressWarnings("ClassWithTooManyMethods") // Many deprecated methods will be removed in future.
```

Comment:
I think it should be “in the future” in [this context](https://jakubmarian.com/in-the-future-vs-in-future-in-british-and-american-english/).